releasenotes/notes/pcf-sidecars-tags-90a54b31bbdacc67.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Adds `sidecar_present` and `sidecar_count` tags on Cloud Foundry containers
```

```suggestion
    Adds ``sidecar_present`` and ``sidecar_count`` tags on Cloud Foundry containers
```
Not 100% sure, but Agent release notes usually use double-backticks for monospace

---

You're right, thanks!